wiki/Best_Of/2015/en.md

```diff
@@ -0,0 +1,2109 @@
+# 2015
+
+**_[Click here to return to the Best of main page](../ "Best Of main page")_**
+
+## General
+
+- First time in the series to enforce that at least one beatmap in the beatmapset must be played and passed to be votable.
+  - As such, it is entirely possible to get an empty list of votable beatmapset.
+- First time in the series to use a weighting system.
+- [News Results](https://osu.ppy.sh/news/138781453998)
+
+## osu!standard
+
+- Official Google Doc of the results: <https://docs.google.com/spreadsheets/d/1l_8ur1YkaboLx_7bQb70Tgkjg8c7ObyAjmJyWDBM1C0/pubhtml#>
+
+|No.|Beatmap ID|Title|Weighted Score|Raw Votes|Total Plays (Voters)|
+|:---:|:---|:---|:---:|---:|---:|
+|1|332532|[Panda Eyes & Teminite - Highscore [Fort]](https://osu.ppy.sh/s/332532/ "Panda Eyes & Teminite - Highscore [Fort]")|1.72|9817|29272|
```

Comment:
:-1: for using square brackets for creator
_(suppressing this for all other instances)_